server/datastore/mysql/migrations/tables/20230328091816_EnsureUniformCollation.go

```diff
@@ -103,14 +270,14 @@ func changeCollation(tx *sql.Tx, charset string, collation string) (err error) {
 	return err
 }
 
-func Up_20230315104937(tx *sql.Tx) error {
+func Up_20230328091816(tx *sql.Tx) error {
```

Comment:
What do you think about leaving the migration name the same? I'm imagining that allows installations that have not yet upgraded to use the new improved migration, while installations that already successfully ran the migration will do nothing.

Reply:
I think my first instinct was to treat migrations as immutable, per the contributor guidelines.

Thinking about this, I think you're right, in this scenario that's the ideal for us. I have renamed the migration to its old name, thanks!